.github/workflows/vercel-preview.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Vercel Preview
+env:
```

Comment:
This section `env` can be removed.

Reply:
this section is defining environment variables necessary for the workflow

https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

Reply:
I don't think top level would work given the secrets are in the environment `vercel` (that get's referenced later on). You can also [move this to inside the job level](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow) instead.